python/tach/interactive/modules.py

```diff
@@ -135,6 +140,7 @@ def _build_subtree(
         root: FileNode,
         depth: int = 1,
         exclude_paths: list[str] | None = None,
+        use_regex_matching: bool | None = None,
```

Comment:
```suggestion
        use_regex_matching: bool | None = True,
```
Let's default to true for now

Reply:
The default value of True if being passed into this function by Rust (where all other defaults are also being set), I don't think we should do it specifically here.